jsx/Form.js

```diff
@@ -145,6 +145,73 @@ FormElement.defaultProps = {
   },
 };
 
+/**
+ * FieldsetElement Component.
+ * React wrapper for <fieldset> element that is nested inside <FormElement></FormElement>,
+ * and accepts children react components. A fieldset groups related elements in a form.
+ *
+ * The form elements can be passed by:
+ * 1. Form components nested directly inside <FieldsetElement></FieldsetElement>
```

Comment:
Can't do a multiline suggest, but.. there's only one thing in this list, why not just make it a sentence?